packages/sh/test/fixtures/445.sh

```
@@ -0,0 +1,2 @@
+curl --location --request PUT 'https://url.com/bla?q=\'hi\'' \
```

Shellcheck (0.10.0)

[warning] Did you forget to close this single quoted string?

(SC1078)